### PR TITLE
2 add aboutxml and other base mod files

### LINKED ISSUE #2: add about.xml and other base mod files

### PR DESCRIPTION
**Pull request by itepastra**
I created a gitignore and about xml file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1.4/Assemblies
+1.4/Source/WWE/.vs
```
Comment (RedHotKomet) [inline .gitignore]: wrong name, dont use WWE, use ATE instead

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+1.4/Source/WWE/bin
```
Comment (RedHotKomet) [inline .gitignore]: wrong name, dont use WWE, use ATE instead

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+1.4/Source/WWE/obj
```
Comment (RedHotKomet) [inline .gitignore]: wrong name, dont use WWE, use ATE instead

File: About/About.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ModMetaData>
+  <name>Android Tiers Evolved</name>
+  <author>RedHotKomet</author>
+  <url></url>
+  <description>
+    Android tiers for 1.4 with a couple of small edits
```
Comment (RedHotKomet) [inline About/About.xml]: edit description pls

**Review by RedHotKomet — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (itepastra) [reply .gitignore]: Fixed in the new gitignore

**Review by RedHotKomet — APPROVED** [review]
noice

File: About/About.xml (hunk 4 above)
Comment (itepastra) [reply About/About.xml]: what should the discription be then?

**Pull request merged**